views/partials/_whats-new.njk

```diff
@@ -6,8 +6,8 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds-from-desktop">
         <h2 id="whats-new" class="govuk-heading-l">What’s new</h2>
-        <p class="govuk-body">16 December 2021: We’ve released GOV.UK Frontend v4.0.0. This ‘breaking release’ contains an update to the accordion and fixes to the footer.</p>
-        <p class="govuk-body"><a href="https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.0" class="govuk-link">Read the release notes to find out what changes you need to make to update</a>.</p>
+        <p class="govuk-body">3 February 2022: We’ve released GOV.UK Frontend v4.0.1. This ‘fix release’ contains improvements to the error summary, input and accordion components.</p>
+        <p class="govuk-body"><a href="https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.1" class="govuk-link">Read the release notes to find out what changes you should make to update</a>.</p>
```

Comment:
I think for fix/feature releases where user's don't need to make changes to upgrade, we've previously said something like this:

```suggestion
        <p class="govuk-body"><a href="https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.1" class="govuk-link">Read the release notes to see what’s changed</a>.</p>
```

Reply:
That's a good point. Could something like this work?

```suggestion
        <p class="govuk-body"><a href="https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.1" class="govuk-link">Read the release notes to learn more about the fixes we've made</a>.</p>
```

Reply:
Sounds good 👍

Reply:
Cheers!